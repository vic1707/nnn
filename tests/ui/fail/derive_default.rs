use nnn::nnn;

#[nnn(derive(Default))]
struct DefaultFloat(f64);

fn main() {}
