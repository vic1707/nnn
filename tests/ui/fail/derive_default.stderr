error: To derive the `Default` trait, use the `default` or `default = ..` argument.
 --> tests/ui/fail/derive_default.rs:3:14
  |
3 | #[nnn(derive(Default))]
  |              ^^^^^^^
